jsonapi_pydantic/v1_0/resource/resource.py

```diff
@@ -11,7 +11,7 @@
 
 Id = Optional[str]
 Attributes = Optional[Dict[str, Any]]
-Relationships = Optional[Dict[str, Relationship]]
+Relationships = Optional[Dict[str, Union[Relationship, List[Relationship]]]]
```

Comment:
`7.2.2.2: The value of the relationships key MUST be an object (a “relationships object”).`

Reply:
Oups again :)
Perhaps you should provide an example of an array of [resource identifier objects](https://jsonapi.org/format/#document-resource-identifier-objects) for non-empty to-many relationships.
For dummy guys like me